docs/src/about_ranking.md

```diff
@@ -26,4 +26,4 @@ Gravity = Decay gravity, 1.8 is default
 
 A plot of rank over 24 hours, of scores of 1, 5, 10, 100, 1000, with a scale factor of 10k.
 
-![](https://i.imgur.com/w8oBLlL.png)
+![](https://i.imgur.com/QukPR03.png)
```

Comment:
This should really use a file from the repo, no need for imgur.

Reply:
That's fair. The pull request actually also places the file in the docs folder but I used imgur under the assumption that there was a reason it was done like that initially.

Reply:
This is good as is, I do prefer imgur links in the markdown because then we can copy-paste that markdown around, and the pictures go with it.

Reply:
Why not host the image on dev.lemmy.ml, or link from gitea if its about that?

Reply:
I'm fine with that, a full github or gitea link would pry be a better option. There's several other pics currently in the docs that'd we'd need to do that for anyway.

Reply:
#655